src/components/picker/PickerDialog.ios.js

```diff
@@ -39,17 +38,6 @@ class PickerDialog extends BaseComponent {
     );
   }
 
-  renderPicker() {
-    const {children, onValueChange, selectedValue, renderNativePicker, pickerStyle, testID} = this.props;
-    if (_.isFunction(renderNativePicker)) {
-      return renderNativePicker(this.props);
-    }
-    return (
-      <WheelPicker style={pickerStyle} selectedValue={selectedValue} onChange={onValueChange} testID={`${testID}.wheelPicker`}>
```

Comment:
Same here, you don't need the `selectedValue` and `onValueChange` props

Reply:
fixed